test/groovy/TestsPublishResultsTest.groovy

```diff
@@ -126,4 +129,45 @@ class TestsPublishResultsTest extends BasePiperTest {
         assertTrue('Cobertura options are not empty', publisherStepOptions.cobertura == null)
         assertTrue('JMeter options are not empty', publisherStepOptions.jmeter == null)
     }
+
+    @Test
+    void testBuildResultStatus() throws Exception {
+        // execute test
```

Comment:
I veto to have these comments in the code. 
I think that should be clear from the code, what the test execution is and which part is the assertion.